Header header
uint32 blob_count
Blob3d[] blobs
